backend/middleware/brands/validations/brandSignUpValidate.js

```diff
@@ -37,6 +37,10 @@ const brandSignUpValidate = async (req, res, next) => {
   if (existingBrand) {
     return res.status(400).json({ message: "Brand already exists" });
   }
+  const nameTaken = await Brand.findOne({name:name})
```

Comment:
seems like name and emailId uniqueness check is taken care here, hence removed the other check from brandController.

Reply:
yes, all checks are added in the validations.